BUILD

```diff
@@ -341,6 +341,13 @@ ALL |= JAVA17
 
 ALL |= JAVA21
 
+COMMIT_SUFFIXED_TAGS = {
+    original + "-{COMMIT_SHA}": target
```

Comment:
@thesayyn I'm not super convinced this is right, is there some way to print out the maps?

Reply:
never mind, it's just a python print huh?